sdk/daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ChoiceAuthorityTest.scala

```diff
@@ -200,15 +185,10 @@ class ChoiceAuthorityTest(majorLanguageVersion: LanguageMajorVersion)
       }
     }
 
-    "change/gain authority {A,B}-->{C} (need A)" in {
-      inside(runExample(theAut = Set(charlie), theGoal = alice)) { case Left(err) =>
-        inside(err) { case SError.SErrorDamlException(FailedAuthorization(_, why)) =>
-          inside(why) { case cma: CreateMissingAuthorization =>
-            cma.authorizingParties shouldBe Set(charlie)
-            cma.requiredParties shouldBe Set(alice)
-          }
-        }
-      }
+    "try to gain authority {A,B}-->{C} (need A)" in {
+      runExample(theAut = Set(charlie), theGoal = alice) shouldBe Left(
+        SError.SErrorDamlException(RejectedAuthorityRequest(Set(alice, bob), Set(charlie)))
```

Comment:
This exception changed? Should `RejectedAuthorityRequest` not be inside a `FailedAuthorization`?